### PR TITLE
Dev mode

### PR DESCRIPTION
**Pull request by NimG98**

### CODE REVIEW

File: finish/system/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<!--
-  Copyright (c) 2016 IBM Corp.
-
+<!-- 
+  Copyright (c) 2017, 2018 IBM Corp.
```
Comment (gkwan-ibm) [inline finish/system/src/main/webapp/index.html]: Why years not `2016, 2020`?

File: README.adoc [hunk 2]
```diff
@@ -445,9 +450,12 @@ guide.
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Create the `kubernetes.yaml`.#
+#Update the `kubernetes.yaml`.#
 `kubernetes.yaml`
 ----
+[role="edit_command_text"]
+Replace `[registry-server]` with your container registry server.
```
Comment (gkwan-ibm) [inline README.adoc]: Can make a hotspot to `[registry-server]`

File: README.adoc [hunk 3]
```diff
@@ -445,9 +450,12 @@ guide.
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Create the `kubernetes.yaml`.#
+#Update the `kubernetes.yaml`.#
 `kubernetes.yaml`
 ----
+[role="edit_command_text"]
+Replace [hotspot=sysImage hotspot=invImage file=0]`[registry-server]` with your container registry server.
+You can get the login server for your registry by running `az acr show -n [registry-name] --query loginServer` command.
```
Comment (Charlotte-Holt) [inline README.adoc]: Update to
"You can get the login server for your registry by running the `az acr show -n [registry-name] --query loginServer` command."

File: README.adoc [hunk 4]
```diff
@@ -585,32 +593,30 @@ The default properties that are defined in the [hotspot file=0]`pom.xml` file ar
...
-| [hotspot=invHttpPort file=0]`inv.service.http.port`    | The HTTP port of the {kube} Service `inventory-service`, 9081 by default.
+| [hotspot=sysHttpPort file=0]`sys.http.port`    | The HTTP port for the {kube} Service `system-service`, 9080 by default.
+| [hotspot=invHttpPort file=0]`inv.http.port`    | The HTTP port of the {kube} Service `inventory-service`, 9081 by default.
 |===
 
 
 === Running the tests
 
-Run the Maven `verify` goal to test your microservices by replacing the `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]`
+Run the Maven `failsafe:integration-test` goal to test your microservices by replacing the `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]`
```
Comment (Charlotte-Holt) [inline README.adoc]: Update to 
"Run the `failsafe:integration-test` Maven goal to test your microservices by replacing `[system-EXTERNAL-IP]` and `[inventory-EXTERNAL-IP]` with the values that were determined in the previous section."

Comment (NimG98) [reply README.adoc]: I think saying Maven first makes more sense.

**Pull request merged**